src/commands/createNewProject/dockerfileSteps/CreateDockerfileProjectStep.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ext } from "@microsoft/vscode-azext-serviceconnector";
+import { AzureWizardExecuteStep, UserCancelledError, nonNullValueAndProp } from "@microsoft/vscode-azext-utils";
+import { validateFuncCoreToolsInstalled } from "../../../funcCoreTools/validateFuncCoreToolsInstalled";
+import { localize } from "../../../localize";
+import { cpUtils } from "../../../utils/cpUtils";
+import { type IFunctionWizardContext } from "../../createFunction/IFunctionWizardContext";
+
+export class CreateDockerfileProjectStep extends AzureWizardExecuteStep<IFunctionWizardContext>{
+    public priority: number = 100;
+
+    public async execute(context: IFunctionWizardContext): Promise<void> {
+        const message: string = localize('installFuncTools', 'You must have the Azure Functions Core Tools installed to run this command.');
```

Comment:
I mentioned this in another PR, but you should front-load this error. Nothing more annoying than going through a project wizard and then having it fail right at the end.